docs/react-v9/contributing/dev-workflow.md

```diff
@@ -100,3 +101,13 @@ Common checklist for PR's
 
 If you're using an internal Microsoft linked account, feel free to squash that big green button. 🎉
 If you're not using a Microsoft linked account, someone from the team will have to merge it for you. Thanks for the contribution! 🙏
+
+### Pulling someone elses branch
+
+Sometimes you might have to pull down someone else's branch to test or work on something. Working with forks makes this process a little different than a standard git set up. The process is similar to the initial upstream master:
+
+```
+git remote add yourFriendsRepo https://github.com/yourFriendsUserName/fluentui.git // creates a remote of your friends fork on your local machine
+git fetch yourFriendsRepo yourFriends/fancy/branch // grabs the branch you want to work on
+git checkout feat/react-drawer/move-scroll-logic-to-context // switches to the branch
+```
```

Comment:
I would say this prone to have some issues. What I do now is use [github cli](https://cli.github.com/), this is what the process looks like:
```sh
git pull upstream master
gh pr checkout <number of PR>
```

This is restricted to PRs, but imo it's easier to ask someone to post a draft PR than have 10+ remotes 😅. Feel free to discard or add it as a second option, just thought I would share 🙂

Reply:
Yeah. Good call, that's what I wanted to do since it's more clearly/Obviously the 'Github way'. But when I tried for Marco's PR it errored on. So this was my 'workaround'